neptune/sessions.py

```diff
@@ -61,9 +61,31 @@ def __init__(self, api_token=None, proxies=None):
         self._client = Client(self.credentials.api_address, self.credentials.api_token, proxies)
 
     def get_project(self, project_qualified_name):
-        """
+        """Get a project with given project qualified name.
+
+        In order to access experiment data one needs to get a `Project` object first.
+        This method gives you the ability to obtain a `Project` object for a project with given project qualified name,
+        ie. project name preceded by the namespace to which this project belongs.
+
+        Args:
+            project_qualified_name(str): A full, qualified project name.
```

Comment:
please document arguments like this:
api_token (:obj:`str`, optional, default is ``None``): some text

you can see three pieces of info : 1) object type, 2) if optional, 3) default value